src/Microsoft.Diagnostics.EventFlow.EtwUtilities/project.json

```diff
@@ -0,0 +1,40 @@
+{
+  "version": "1.0.0-*",
+
+  "dependencies": {
+    "NETStandard.Library": "1.6.0"
```

Comment:
Excellent thought, but unfortunately netstandard1.3 requires 4.6 and we depend on System.Diagnostics.Process which is not available for netstandard1.2 (which can do 451). Bottom line this is probably best we can do at the moment.